.storybook/main.ts

```diff
@@ -41,11 +41,6 @@ const config: StorybookConfig = {
     viteFinal: async (config, {configType}) => {
         return mergeConfig(config, {
             ...viteConfig,
```

Comment:
This `viteConfig` comes from `dev/vite.config.ts`.  Maybe it should move to the repo root. 🤔